cmd/s2i/main.go

```diff
@@ -171,6 +171,7 @@ $ s2i build . centos/ruby-22-centos7 hello-world-app
 	buildCmd.Flags().StringVarP(&(cfg.Ref), "ref", "r", "", "Specify a ref to check-out")
 	buildCmd.Flags().StringVarP(&(cfg.AssembleUser), "assemble-user", "", "", "Specify the user to run assemble with")
 	buildCmd.Flags().StringVarP(&(cfg.ContextDir), "context-dir", "", "", "Specify the sub-directory inside the repository with the application sources")
+	buildCmd.Flags().StringVarP(&(cfg.ExcludeRegExp), "exclude", "", "", "Regular expression for selecting files from the source tree to exclude from the tar stream; see https://golang.org/pkg/regexp for syntax")
```

Comment:
they don't have to write complex regex.  --exclude="" will be the primary use case for users who want to include git (at least that better work :)  if it matches everything that's a problem)